diff -dur kgicon.orig/kgi/accel/S3/virge.c kgicon/kgi/accel/S3/virge.c
--- kgicon.orig/kgi/accel/S3/virge.c	Sat Jul 10 07:18:56 1999
+++ kgicon/kgi/accel/S3/virge.c	Sun Jul 11 15:30:11 1999
@@ -46,7 +46,7 @@
 /*
 ** The ViRGE accelleration stuff
 */
-
+/*
 static int ViRGE_write_acceleration_registers (kgiu8 n) {
   int addr;
   kgiu32 i;
@@ -62,6 +62,7 @@
   }
   return -EOK;
 }
+*/
 
 inline static int ViRGE_copy_box (struct kgi_cbox *b,struct kgi_display *dpy) {
   DEBUG1("ViRGE: Accellerated CopyBox");
Only in kgicon/kgi/accel/S3: virge.c~
